libraries/server/library/src/main/java/com/chesire/nekome/library/api/LibraryEntity.kt

```diff
@@ -0,0 +1,29 @@
+package com.chesire.nekome.library.api
+
+import com.chesire.nekome.core.flags.SeriesStatus
+import com.chesire.nekome.core.flags.SeriesType
+import com.chesire.nekome.core.flags.Subtype
+import com.chesire.nekome.core.flags.UserSeriesStatus
+import com.chesire.nekome.core.models.ImageModel
+
+/**
+ * Entity related to a singular library item.
+ */
+data class LibraryEntity(
```

Comment:
Maybe LibraryDomain instead? Since it isn't an entity, and isn't used to transfer between app and server